app/services/payment/models/key.rb

```diff
@@ -0,0 +1,41 @@
+require 'addressable/template'
+require 'json'
+
+module Payment
+  module Models
+    class Key < Client
+      include Initializable
+
+      attr_accessor :id, :name, :merchant, :secret_key, :created_at, :expiry
+
+      # For more information about how these URI templates are structured read the explaination in the RFC
+      # https://www.rfc-editor.org/rfc/rfc6570.txt
+      PATH = Addressable::Template.new("v1/merchants/{publisher_id}/keys{/id}")
```

Comment:
Extra slash in the parameter?

Reply:
Don't think so, Faraday should trim extra slashes 😄